### PR TITLE
Configure extensible SDK

### PR DESCRIPTION
**Pull request by mythi**
These two commits are needed to configure eSDK properly.

Not fully tested yet, submitting a PR for others to review.

@kad @okartau

### CODE REVIEW

**Comment by kad** [comment]
looks good, but don't we want to provide some default ? Or I'm missing something.

**Comment by mythi** [comment]
@kad default for sdk update url? I wanted to give that through examples. The only place it goes to is devtool.conf which can easily be edited afterwards as well.

**Comment by kad** [comment]
ok. but are those examples end-up in sample devtool.conf that is shipped in installer ?
If yes, then I'm ok. but if not, it would be better to have some pre-defined URL.

**Comment by mythi** [comment]
@kad good point. Currently we are missing the 'latest' symlink so what'd be the proper pre-defined URL?

**Comment by kad** [comment]
@okartau supposed to restore latest symlinks. So pointing to latest is ok

**Comment by mythi** [comment]
rebased, added more fixes and force pushed.

setting 'test-build' label.

**Comment by mythi** [comment]
rebased, blacklisted two more components, and force pushed

**Comment by mythi** [comment]
rebased, updated/split commits, and force pushed

File: meta-ostro/conf/distro/ostro.conf [hunk 1]
```diff
@@ -4,23 +4,34 @@ DISTRO_VERSION = "1.0~snapshot-${DATE}"
...
+# SDK_UPDATE_URL defines the eSDK updateserver configured in
+# devtool.conf when populate-sdk-ext is run.
+#
+# Example SDK_UPDATE_URLs are:
+# https://download.ostroproject.org/builds/ostro-os/<build>/<machine> OR
+# https://download.ostroproject.org/releases/ostro-os/milestone/<release>/<machine>
+#
+# The defaultsetting can be overridded in local.conf
+OSTRO_SDK_UPDATE_URL ??= "https://download.ostroproject.org/builds/ostro-os/latest/${MACHINE}"
+
```
Comment (kad) [inline meta-ostro/conf/distro/ostro.conf]: sdk-ext missing before $MACHINE

File: meta-ostro/recipes-kernel/bcm43340/bcm43340-mod.bbappend [hunk 2]
```diff
@@ -0,0 +1 @@
+COMPATIBLE_MACHINE="edison"
```
Comment (kad) [inline meta-ostro/recipes-kernel/bcm43340/bcm43340-mod.bbappend]: should those be here ?

File: meta-ostro/conf/distro/ostro.conf (hunk 1 above)
Comment (mythi) [inline meta-ostro/conf/distro/ostro.conf]: On Thu, Mar 31, 2016 at 2:43 AM, Alexander D. Kanevskiy <
notifications@github.com> wrote:

> In meta-ostro/conf/distro/ostro.conf
> <https://github.com/ostroproject/meta-ostro/pull/63#discussion_r57982027>:
>
> > -		${OSTRO_PACKAGE_FEED_BUILDID}/\
> > -		${OSTRO_SDK_EXT_DIR}/\
> > -		${MACHINE}\
> > -"
> > +SDK_UPDATE_URL = "${OSTRO_SDK_UPDATE_URL}"
> > +
> > +# SDK_UPDATE_URL defines the eSDK updateserver configured in
> > +# devtool.conf when populate-sdk-ext is run.
> > +#
> > +# Example SDK_UPDATE_URLs are:
> > +# https://download.ostroproject.org/builds/ostro-os/<build>/<machine> OR
> > +# https://download.ostroproject.org/releases/ostro-os/milestone/<release>/<machine>
> > +#
> > +# The defaultsetting can be overridded in local.conf
> > +OSTRO_SDK_UPDATE_URL ??= "https://download.ostroproject.org/builds/ostro-os/latest/${MACHINE}"
> > +
>
> sdk-ext missing before $MACHINE
>
>
Nice catch! Thanks

File: meta-ostro/recipes-kernel/bcm43340/bcm43340-mod.bbappend (hunk 2 above)
Comment (mythi) [inline meta-ostro/recipes-kernel/bcm43340/bcm43340-mod.bbappend]: On Thu, Mar 31, 2016 at 2:44 AM, Alexander D. Kanevskiy <
notifications@github.com> wrote:

> In meta-ostro/recipes-kernel/bcm43340/bcm43340-mod.bbappend
> <https://github.com/ostroproject/meta-ostro/pull/63#discussion_r57982143>:
>
> > @@ -0,0 +1 @@
> > +COMPATIBLE_MACHINE="edison"
>
> should those be here ?
>
>
No. Just to test build/testing purposes.

**Comment by mythi** [comment]
@kad final version, force pushed.

**Comment by kad** [comment]
:+1: looks good.

**Comment by kad** [comment]
retest this please

**Comment by mythi** [comment]
rebase + force pushed.

**Comment by mythi** [comment]
rebased + force pushed

**Comment by mythi** [comment]
rebased, added 545e135, and force pushed

@kad can you take a look  545e135

**Comment by mythi** [comment]
rebased, resolved conflicts due to swupd, and force pushed

**Comment by mythi** [comment]
test this please

"Slave went offline"

**Comment by mythi** [comment]
Needs #97 so restarting.

test this please

**Pull request merged**